hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/scm/TestStorageContainerManager.java

```diff
@@ -311,7 +324,7 @@ public void testBlockDeletionTransactions() throws Exception {
         OzoneTestUtils.closeContainers(keyInfo.getKeyLocationVersions(),
             cluster.getStorageContainerManager());
       }
-
+      Thread.sleep(3000);
```

Comment:
Can use this method to wait for the Container to close but need get the Container list first.
https://github.com/apache/ozone/blob/de76edc0e1ecb4516b0232e21336b889d5d1081f/hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/container/TestHelper.java#L326-L327

Reply:
@xichen01  , After I updated configs for MiniOzoneCluster, this Thread.sleep is not needed for waiting to get containers closed. After removing the sleep, I re-ran 500 times repeated runs for this test class and all passwed 100%. Here is the green CI [link](https://github.com/devmadhuu/ozone/actions/runs/7540060163).

Pls re-review.